docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,39 @@
+[[type-definitions-schema-configuration]]
+= Schema Configuration
+
+Neo4j GraphQL Library supports several functionalities such as CRUD operations, aggregation, filtering, and others.
+To make them work, a large amount of GraphQL types are generated.
+
+In some cases, it may be advisable to reduce the scope of the API produced. 
+This section provides information on how to reduce the size of the schema, improve the performance, and limit access to unwanted operations.
```

Comment:
I'm slightly nervous about this line because I don't want users to expect massive performance gains with this as they aren't likely to be very noticeable unless they extensively use these features on a large schema. What do you think about this new wording?

```suggestion
This section provides information on how to limit access to unwanted operations and reduce the size of the schema which can improve the performance.
```